<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Send_Email</class>
 <widget class="QWidget" name="Send_Email">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1024</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pacifico</family>
     <pointsize>34</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Your Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>400</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pacifico</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Send It!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>824</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pacifico</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>your.email@example.com</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_skip">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pacifico</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Nevermind,
changed my mind.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
